### PR TITLE
Lm/tc039 change appearance organization folder

### PR DESCRIPTION
**Pull request by li-ma1**
tc039 change appearance organization folder

### CODE REVIEW

**Comment by li-ma1** [comment]
[AT_039.001| Organization Folder > Change Appearance to Default Icon](https://trello.com/c/SYDgBdeS/161-at039001-organization-folder-change-appearance-to-default-icon)

**Review by EugeniaSrednitskaya — APPROVED** [review]

File: src/test/java/CreateOrganizationFolderTest.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import runner.BaseTest;
+
```
Comment (RusAli) [inline src/test/java/CreateOrganizationFolderTest.java:6]: white space

File: src/test/java/ChangeAppearanceToDefaultIconOrganizationFolderTest.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import runner.BaseTest;
+
```
Comment (RusAli) [inline src/test/java/ChangeAppearanceToDefaultIconOrganizationFolderTest.java:6]: white space

File: src/test/java/CreateOrganizationFolderTest.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    }
+
+    @Test
+    public void testCreateOrganizationFolder () {
+
+        getDriver().findElement(By.linkText("New Item")).click();
+        getDriver().findElement(By.id("name")).sendKeys(VALID_VALUE_FOR_NAME);
+        getDriver().findElement(By.className("jenkins_branch_OrganizationFolder")).click();
+        getDriver().findElement(By.id("ok-button")).click();
+        getDriver().findElement(By.id("yui-gen17-button")).click();
```
Comment (RusAli) [inline src/test/java/CreateOrganizationFolderTest.java:26]: u can create separate method to create folder

**Review by RusAli — APPROVED** [review]

**Pull request merged**